manifests/init.pp

```diff
@@ -23,11 +23,12 @@
 #   Defaults to ['tty0', 'tty1', 'tty2', 'tty3', 'tty4']
 #
 #   * If set to false, management of /etc/securetty will be disabled
-#   * If set to true or an empty array, root will not be able to log into
-#     any local device
+#   * If the Array is empty (default) then root will not be able to log into
```

Comment:
securetty can take a value of true and it's effects have been removed from the docs. Why?